patches/react-native-pull+2.0.2.patch

```diff
@@ -4,199 +4,199 @@ index 0000000..f72e93a
 --- /dev/null
```

Comment:
Why you are changing this file?

Reply:
It's caused by the merge of the branch with the master made with Github Desktop, I'm trying to fix it.

Reply:
@fpersico Fixed!